styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
```

Comment:
@amberRrucker you will see here and below that I refactored the rug to use a few modular classes to accomplish the desired results. Please feel free to adjust it further for aesthetics.

The key lesson here is the use of single-depth class selectors that do not conflict with any other styles, do not extend any other styles, and express their usage in a semantic fashion. Such class design fits in very well with the classes provided by Bootstrap.